irepela/homework_5/backward_each_word.py

```diff
@@ -0,0 +1,12 @@
+def backward_each_word(text: str) -> str:
+    """
+        Reverses each word in given string
+
+        Args:
+            text (str): string to reverse word in
+
+        Returns:
+            str: string with reversed words
+    """
+
+    return " ".join([word[::-1] for word in text.split(" ")])
```

Comment:
if you use list comprehension you immediatly unpack all into intermediate list, 
better here to use generator expression.

```
return " ".join((word[::-1] for word in text.split(" ")))


and since it is the only input parameter, parenthesis can be avoided

 return " ".join(word[::-1] for word in text.split(" "))
```